com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeGIBaking.cs

```diff
@@ -240,6 +252,7 @@ static void BrickCountInDirections(out Vector3Int cellsInXYZ, float brickSizeInM
 
         // NOTE: This is somewhat hacky and is going to likely be slow (or at least slower than it could).
         // It is only a first iteration of the concept that won't be as impactful on memory as other options.
+#if false // TODO: Make sure we intend to keep this before fixing
```

Comment:
This is currently unused, so I didn't update it. Either I fix it or we remove it.

Reply:
It is going to be used, we just need to discuss the UX for it. 

Also we use it while debugging (it is just a matter of uncommenting out one single line in code and it will pop up in the UI).

We need this to be functional .